promoter/image/promoter.go

```diff
@@ -97,6 +98,8 @@ type promoterImplementation interface {
 // PromoteImages is the main method for image promotion
 // it runs by taking all its parameters from a set of options.
 func (p *Promoter) PromoteImages(opts *options.Options) (err error) {
+	logrus.Infof("kpromo[%d]: PromoteImages start", time.Now().Unix())
```

Comment:
Logrus is already able to log timestamps, we just have to set a different formatter: https://github.com/sirupsen/logrus/blob/f8bf7650dccb756cea26edaf9217aab85500fe07/text_formatter.go#L48-L50